### ISSUE TITLE
Internship plan for [Tega]

### ISSUE THREAD

**Issue by chendelin1982** [Websoft9/websoft9#525]
Intership use [Github issues](https://github.com/Websoft9) for task management.

- Start time: September 2nd, 2024
- End time:  

### Technical Tasks

- [ ] Documentation translation
- [ ] Create docker compose library
- [ ] Create Terraform templates for installing Websoft9 in major Cloud

### Daily Operational

- [ ] Build image
- [ ] Marketing contents

### Team study and presentation

- [ ] [Stye Guide](https://docs.gitlab.com/ee/development/documentation/)
- [ ] [Docker Compose](https://docs.docker.com/compose/)
- [ ] [Terraform](https://www.terraform.io/)

## Checkpoint

- [ ] A Daily Check-In With The Intern
- [ ] Weekly Meeting/Video Call With The Intern
- [ ] Weekly Project Checkpoint
- [ ] Pre-Program Evaluation 
- [ ] End-of-Program Evaluation

**Comment by qiaofeng1227** [comment]
Documentation translation Check points:

1. comply with the styleguide of the documentation
2. accuracy in the use of English words
3. documentation and App Consistency
4. simple errors in formatting syntax

Refer to:
[docs url](https://support.websoft9.com/en/docs/next/apps/)
[Style Guide](https://support.websoft9.com/en/docs/next/document)
[translate tool](https://www.deepl.com/zh/translator)
[Websoft9 Hosting platform](http://t.demo.goweb.cc:9000/)

**Comment by qiaofeng1227** [comment]
## 2024  September week1 work plan：

1. Documentation translation(improve)（10apps）

Documentation translation Check points:
- comply with the styleguide of the documentation
- accuracy in the use of English words
- documentation and App Consistency
- simple errors in formatting syntax
Refer to:
[docs url](https://support.websoft9.com/en/docs/next/apps/)
[Style Guide](https://support.websoft9.com/en/docs/next/document)
[translate tool](https://www.deepl.com/zh/translator)
[Websoft9 Hosting platform](http://t.demo.goweb.cc:9000/)

repository: https://github.com/Websoft9/doc.websoft9.com/
example source file: [zookeeper Corresponding document source file](https://github.com/Websoft9/doc.websoft9.com/blob/main/i18n/en/docusaurus-plugin-content-docs/current/apps/zookeeper.md)

Once the fix is complete, submit the pull_request to doc.websoft9.com's dev brachch

- [x] vault
- [x]  vaultwarden
- [x]  vtigercrm
- [x]  woocommerce
- [x]  wordpress
- [x]  xwiki
- [x]  zabbix
- [x]  zammad
- [x]  zentao
- [x]  zookeeper

**Comment by qiaofeng1227** [comment]
## 2024  September week2 work plan：

1. improve documentation

- [x] thingsboard
- [x] tinyproxy
- [x] tooljet
- [x] traefik
- [x] trivy
- [x] typesense
- [x] typo3
- [x] umami
- [x] umbraco
- [x] uptimekuma

2. Learn docker and docker-compose, write your own docker-compose.yml, and run an arbitrary sample webapp. OK

**Comment by qiaofeng1227** [comment]
## 2024  September Week-3 work plan：

1. improve documentation

- [x] sqlserver
- [x] srs
- [x] sscms
- [x] strapi
- [x] suitcrm
- [x] superset
- [x] syncthing
- [x] teamcity
- [x] teleport
- [x] tensorflow

2. Write a docker-compose.yml of **immich** app
refer to: [immich official dock-compose ](https://immich.app/docs/install/docker-compose/),[websoft9 rule](https://github.com/Websoft9/docker-library/blob/main/docs/code_owner.md)

If you complete this app, please pull request to [docker-library](https://github.com/Websoft9/docker-library)

3. Write a [Terraform](https://www.terraform.io/) example which can be create a ESC by websoft9 product
Detail refer to: https://github.com/Websoft9/CloudForms/issues/3

**Comment by qiaofeng1227** [comment]
## 2024  September Week-4 work plan：

1. improve documentation

- [x] prestashop
- [x] prometheus
- [x] pydio
- [x] rabbitmq
- [x] redash
- [x] redis
- [x] redisinsight
- [x] registry
- [x] rethinkdb
- [x] rocketchat
- [x] rocketmq
- [x] rundeck
- [x] scratch
- [x] screego
- [x] seafile
- [x] semaphore
- [x] snapdrop

2. write docker-compose for [Gladys](https://github.com/Websoft9/docker-library/issues/706)

**Comment by qiaofeng1227** [comment]
## 2024  October Week-5 work plan：

1. improve documentation

- [x] postgrest
- [x] postgresql
- [x] portainer
- [x] pmm
- [x] plausible
- [x] phpmyadmin
- [x] pgadmin
- [x] penpot
- [x] passbolt
- [x] owncloud
- [x] opensearch

2. write docker-compose for [mlflow](https://github.com/Websoft9/docker-library/issues/711)，[milvus](https://github.com/Websoft9/docker-library/issues/709)

**Comment by qiaofeng1227** [comment]
## 2024  October Week-6 work plan：

1. improve documentation

- [x] nocobase
- [x] nocodb
- [x] nodered
- [x] nopcommerce
- [x] odoo
- [x] onlyoffice
- [x] onlyofficedocs

2. write docker-compose for [weaviate](https://github.com/Websoft9/docker-library/issues/710)，[DocuSeal](https://github.com/Websoft9/docker-library/issues/719)

**Comment by qiaofeng1227** [comment]
## 2024  October Week-7 work plan：

1. improve documentation

- [x] nexus
- [x] nextterminal
- [x] nextcloud
- [x] netdata
- [x] neo4j
- [x] navidrome
- [x] n8n

2. write docker-compose for [vespa](https://github.com/Websoft9/docker-library/issues/708)

**Comment by qiaofeng1227** [comment]
## 2024  October Week-8 work plan：

1. improve documentation

- [x] danswer
- [x] mage
- [x] pmm
- [x] photoprism
- [x] portkey
- [x] semaphore
- [x] zulip
- [x]  nginx


2. write docker-compose for [ckan](https://github.com/Websoft9/docker-library/issues/319)

**Comment by qiaofeng1227** [comment]
2024 October Week-9 work plan：
1. [new task](https://github.com/Websoft9/CloudForms/issues/5)
2. write docker-compose for [ckan](https://github.com/Websoft9/docker-library/issues/319)

**Comment by qiaofeng1227** [comment]
2024 October Week-10 work plan：
1. [php dockerfile](https://github.com/Websoft9/docker-library/issues/916)

**Comment by qiaofeng1227** [comment]
2024  Week-11 work plan：
1. [Llama 3](https://github.com/Websoft9/docker-library/issues/943)

**Comment by qiaofeng1227** [comment]
2024 October Week-12 work plan：
1. [github action](https://github.com/Websoft9/docker-library/issues/916)
